entrypoint.sh

```diff
@@ -10,7 +10,11 @@ fi
 # Download Minecraft client .jar (Contains textures used by Minecraft Overviewer)
 CLIENT_URL=$(python3 /home/minecraft/download_url.py "$MINECRAFT_VERSION")
 echo "Using Client URL $CLIENT_URL."
-wget -N "${CLIENT_URL}" -O "${MINECRAFT_VERSION}.jar" -P /home/minecraft/.minecraft/versions/${MINECRAFT_VERSION}/
+wget -N "${CLIENT_URL}" -O "${MINECRAFT_VERSION}.jar" -P /home/minecraft/.minecraft/versions/"${MINECRAFT_VERSION}"/
```

Comment:
I'm not sure if the quoting here will work as expected; I'd need to check. I think something like this may be more clear:
```suggestion
wget -N "${CLIENT_URL}" -O "${MINECRAFT_VERSION}.jar" -P "/home/minecraft/.minecraft/versions/${MINECRAFT_VERSION}/"
```

Reply:
I got it from Shellcheck here: https://github.com/koalaman/shellcheck/wiki/SC2086

I think your suggestion is cleaner.

Reply:
Does Shellcheck still warn on that line? 

I should add some linting to this project 🤔and some better tests 🤔

Reply:
Nope. Now that it's somewhere inside quotes it shouldn't have issues with hypothetical MC versions that have spaces or other IFS stuff in the name.